packages/lib/constants.ts

```diff
@@ -8,7 +8,7 @@ export const WEBAPP_URL =
   RAILWAY_STATIC_URL ||
   HEROKU_URL ||
   RENDER_URL ||
-  "http://localhost:3000";
+  "http://app.cal.local:3000";
```

Comment:
Had to use a different URL than "localhost" to play with cookie sessions between different subdomains. We should make everyone use this new scheme by default, it's needed to make orgs work locally.